src/diagonal.works/b6/cmd/b6/b6.go

```diff
@@ -33,6 +33,7 @@ func main() {
 	readOnlyFlag := flag.Bool("read-only", false, "Prevent changes to the world")
 	staticFlag := flag.String("static", "src/diagonal.works/b6/cmd/b6/js/static", "Path to static content")
 	jsFlag := flag.String("js", "src/diagonal.works/b6/cmd/b6/js", "Path to JS bundle")
+	staticV2Flag := flag.String("static-v2", "frontend/dist", "Path to V2 static content")
```

Comment:
```suggestion
	staticV2Flag := flag.String("static-v2", "frontend/dist", "Path to V2 static content")
	storybookFlag := flag.String("storybook", "frontend/storybook-static", "Path to V2 Storybook")
```

Following along the pattern, would guess something like this so we can point to the static builid of the storybook to later serve it as well